config/ftbquests/quests/chapters/occultism.snbt

```diff
@@ -1040,29 +1157,21 @@
 			]
 		}
 		{
+			title: "Summon Possessed Enderman"
 			icon: "minecraft:ender_pearl"
 			x: 5.0d
 			y: 0.0d
 			shape: "hexagon"
+			description: ["The possessed Enderman is a bit stronger than normal one, but it guarantees Ender Pearl drop!"]

```

Comment:
than *a* normal one, but it *is guaranteed to drop Ender Pearls* (or to drop an Ender Pearl)